index.bs

```diff
@@ -396,8 +399,9 @@ Otherwise, do nothing.
 <h4 id="count" oldids="count-label,dom-console-count" method for="console">count(<var>label</var>)</h4>
 
 <emu-alg>
-  1. Let _called_ be the number of times _count_ has been invoked (including this invocation) with
-  the provided _label_.
+  1. Let map be the associated count map.
+  1. If _map_[_label_] <a for="map">exists</a> in <a>count map</a>, <a for="map">set</a> the _value_ of _map_[_label_] to be _map_[_label_] + 1.
```

Comment:
"value" is not a variable, so should not be italic. You can just do "set _map_[_label_] to _map_[_label_] + 1"